modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-aws-load-balancer-with-outposts_{context}"]
+= AWS Load Balancer Operator and AWS Outposts
+
+AWS Load Balancer Operator can work with AWS Outposts. Ingress resources must be annotated with the either regular subnets or outposts subnets (but not both).
+
+.Procedure
+
+* Configure the Ingress resource to use a specified subnet:
++
+.Example Ingress resource configuration
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: echoserver
+  annotations:
+    alb.ingress.kubernetes.io/subnets: <outposts-subnet-id> # <1>
```

Comment:
suggestion to use underscores per [doc guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command:~:text=System%20or%20software%20variable%20to%20be%20replaced%20by%20the%20user) examples:
```suggestion
    alb.ingress.kubernetes.io/subnets: <outposts_subnet_id> # <1>
```

Reply:
ty